### ISSUE TITLE
[bridge] Ignored Error Message

### ISSUE THREAD

**Issue by svenefftinge** [gitpod-io/gitpod#12355]
This [error event](https://console.cloud.google.com/logs/query;cursorTimestamp=2022-08-24T18:45:44.984Z;query=%22410bb5a4-af36-432d-bab1-7f62fd304041%22%0Atimestamp%3D%222022-08-24T18:45:44.984Z%22%0AinsertId%3D%22kf4q9gw5t15i2b1c%22?project=gitpod-191109) is not identified as an error and therefore ws-manager-bridge doesn't flag the workspace as erroneous. As a result users try to restart the workspace, but it doesn't have a backup (because the failure happened on start). 

The error event should result in a proper error message flagging the workspace as broken.

**Comment by geropl** [comment]
@svenefftinge We're ignoring it for a reason: It's part of the lifecycle of a bunch of "good" workspaces that we see a temporary `failled`. I noticed that when I was adding exactly that check... :grimacing: Before we can change anything here we need to reliably fix the problem on workspace side.
Last time I raised it (that we see a lot of conditions only temporarily, and that they are not reliably persisted) I was deferred to `ws-manager-mkii`. :shrug:
